### ISSUE TITLE
Proton won't launch any games using NVIDIA 390 drivers

### ISSUE THREAD

**Issue by mark-raymond** [ValveSoftware/Proton#5597]
# Compatibility Report
- Name of the game with compatibility issues: Everything I've tried; so far A Raven Monologue, Terraforming Mars, and Ticket to Ride
- Steam AppID of the game: 744810, 800270, 108200

## System Information
- GPU: NVIDIA GeForce GT 650M
- Driver/LLVM version: nvidia-driver-390 390.144-0ubuntu0.20.04.1
- Kernel version: 5.13.0-30-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/mark-raymond/971ec8b3079b46d9ab3b764aa0fb4b82
- Proton version: I have mostly tested using 7.0-1, but also tried experimental (7.0-20220218b) and 6.3-8

## I confirm:
- ~[ ] that I haven't found an existing compatibility report for this game.~ N/A, this is not game specific
- [x] that I have checked whether there are updates for my system available.

```
======================
Proton: 1644626092 proton-7.0-1
SteamGameId: 744810
Command: ['/home/mark/.local/share/Steam/steamapps/common/A Raven Monologue/A Raven Monologue.exe']
Options: {'forcelgadd'}
depot: 0.20220131.43
pressure-vessel: 0.20220131.0
scripts: v0.20211029.0-0-g2d89be4
soldier: 0.20220131.0 soldier 0.20220131.0
Kernel: Linux 5.13.0-30-generic #33~20.04.1-Ubuntu SMP Mon Feb 7 14:25:10 UTC 2022 x86_64
======================
```
Other games result in an identical log, with only the game ID and command changing.

## Symptoms

When I run any game:
- I press play
- This dialogue appears:
  ![image](https://user-images.githubusercontent.com/2951053/155016421-8e2e5864-88e9-4982-8ff9-50b16ad3a9a1.png)
- After a few seconds, it disappears. The play button is now a stop button.
- A few seconds later, the stop button returns to being a play button.

## Reproduction

Attempt to run any game using Proton. Obviously this works for most people, so I feel a bit silly filing this bug report, but every other resource I've found with these symptoms has been due to people using NTFS, and I'm not using NTFS, I'm was using BTRFS, and I tried ext4 as well.

**Comment by kisak-valve** [comment]
Hello @mark-raymond, the NVIDIA 390 series video driver is older that the [support cutoff](https://github.com/ValveSoftware/Proton/wiki/Requirements#directx-11-games) for Proton. Your system is trying to use the Intel Ivybridge chipset for Vulkan, but mesa/ANV is not able to render when the X session is being run on the NVIDIA proprietary driver. Additionally, the video driver for Intel's Ivybridge generation provides experimental, incomplete support for Vulkan.

Your Kepler generation NVIDIA chipset exists in a fairly awkward historical position, where NVIDIA dropped support for the laptop variants of that generation after the 418 series proprietary driver. That said, some people have had success using up to the 470 series NVIDIA driver on Kepler laptops, which is the last proprietary driver series to support the desktop variant of the generation.

Please update your video driver.

**Comment by mark-raymond** [comment]
Thank you! I have updated to NVIDIA 470 drivers, and (a) my OS is still working and (b) games are launching, so things are looking good :slightly_smiling_face:

**Comment by mark-raymond** [comment]
Oh, and in case someone else stumbles upon this issue, I'll record what I found along the way:

- Ubuntu 20.04 doesn't have drivers in its repositories between 390 and 470. All the packages in between are "transitional packages" that just point forwards, and would actually install the 470 driver.
- So, as 418 is the last driver officially supporting my graphics card, I initially tried installing 418 from the NVIDIA website (after uninstalling the 390 drivers from the repositories, and disabling nouveau driver), but this didn't work, because it couldn't compile the kernel module. This was due to an incompatibility with kernel 5.11+, which [NVIDIA fixed in the 460 driver](https://www.nvidia.co.uk/Download/driverResults.aspx/171403/en-uk).
- I then tried installing the 470 driver from the repository, and everything just worked.
- If that hadn't, I would have probably tried downgrading my kernel (20.04 originally came with 5.4, and that's still supported) and tried the 418 driver direct from NVIDIA again. But thankfully that wasn't necessary :slightly_smiling_face:
